src/common/models/models_md3.cpp

```diff
@@ -373,9 +373,19 @@ void FMD3Model::RenderFrame(FModelRenderer *renderer, FGameTexture * skin, int f
 		}
 
 		renderer->SetMaterial(surfaceSkin, false, translation);
-		renderer->SetupFrame(this, surf->vindex + frameno * surf->numVertices, surf->vindex + frameno2 * surf->numVertices, surf->numVertices);
+		renderer->SetupFrame(this, surf->vindex + frameno * surf->numVertices, surf->vindex + frameno2 * surf->numVertices, surf->numVertices, {});
 		renderer->DrawElements(surf->numTriangles * 3, surf->iindex * sizeof(unsigned int));
 	}
 	renderer->SetInterpolation(0.f);
 }
 
+//===========================================================================
+//
+//
+//
+//===========================================================================
+
+const TArray<VSMatrix>* FMD3Model::AttachAnimationData()
+{
+	return {};
```

Comment:
Do you want to return an empty array or a null pointer? This does compile down to a null pointer, but then you should use nullptr, not {}.

Reply:
Additionally,this same method has been redundantly added to all existing model,types. It should be put in the base class instead and removed from the subclasses.